src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLQueryTypeImpl.java

```diff
@@ -1121,7 +1122,7 @@ public DataFetcher<Connection<NearbyStop>> stopsByRadius() {
   public DataFetcher<Iterable<FareRuleSet>> ticketTypes() {
     return environment -> {
       Map<FareType, Collection<FareRuleSet>> fareRules =
```

Comment:
Could you do an `instanceof GtfsFareService service` check here and return an empty list otherwise?

Reply:
Background: this API is also used by users that have their own fare calculators.

Reply:
I decided to return the `DefaultFareService` 's fare rules in the case that the service is not a `GtfsFaresService` in order to keep existing functionality but the type check should be there now